virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -70,6 +80,60 @@ include::modules/virt-aws-bm.adoc[leveloffset=+2]
 * xref:../../virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc#virt-connecting-vm-to-ovn-secondary-network[Connecting a virtual machine to an OVN-Kubernetes secondary network]
 * xref:../../virt/vm_networking/virt-exposing-vm-with-service.adoc#virt-exposing-vm-with-service[Exposing a virtual machine by using a service]
 
+// Hiding in ROSA/OSD - todo: double check this
+ifndef::openshift-rosa,openshift-dedicated[]
+[id="ibm-z-linuxone-compatibility_{context}"]
+=== {ibm-z-title} and {ibm-linuxone-title} compatibility
+
+You can use {VirtProductName} in an {product-title} cluster that is installed in a logical partition (LPAR) on an {ibm-z-name} or {ibm-linuxone-name} (s390x architecture) system.
+
+ifdef::openshift-enterprise[]
+:FeatureName: Using {VirtProductName} in a cluster deployed on s390x architecture
+include::snippets/technology-preview.adoc[]
+:!FeatureName:
+endif::[]
+
+Some features are not currently available on s390x architecture, while others require workarounds or procedural changes. These lists are subject to change.
+
+[discrete]
+[id="currently-unavailable-ibm-z_{context}"]
+==== Currently unavailable features
+
+The following features are not available or do not function on s390x architecture:
+
+* Memory hot plugging and hot unplugging
+* Watchdog devices
+* Node Health Check Operator
+* SR-IOV Operator
+* virtual Trusted Platform Module (vTPM) devices
+* {pipelines-title} tasks
+* UEFI mode for VMs
+* PCI passthrough
+* USB host passthrough
+* Configuring virtual GPUs
+* {VirtProductName} cluster checkup framework
+
+[discrete]
+[id="functionality-differences_{context}"]
+==== Functionality differences
+
+The following features are available for use on s390x architecture but function differently or require procedural changes:
+
+* When xref:../../virt/managing_vms/virt-delete-vms.adoc#virt-delete-vm-web_virt-delete-vms[deleting a virtual machine by using the web console], the *grace period* option is ignored.
+
+* When xref:../../virt/managing_vms/advanced_vm_management/virt-configuring-default-cpu-model.adoc#virt-configuring-the-default-cpu-model_virt-configuring-default-cpu-model[configuring the default CPU model], the `spec.defaultCPUModel` value is `"gen15b"` for an {ibm-z-title} cluster.
+
+* When xref:../../virt/vm_networking/virt-hot-plugging-network-interfaces.adoc#virt-hot-plugging-network-interfaces[hot plugging secondary network interfaces], the `virtctl migrate <vm_name>` command does not migrate the VM. As a workaround, restart the VM by running the following command:
++
+[source,terminal]
+----
+$ virtctl restart <vm_name>
+----
+
+* When xref:../../virt/monitoring/virt-exposing-downward-metrics.adoc#virt-configuring-downward-metrics_virt-exposing-downward-metrics[configuring a downward metrics device], the `spec.preference.name` value must be set to `rhel.s390x`.
```

Comment:
I think this should be `rhel.9.s390x` see [common instance types](https://github.com/kubevirt/common-instancetypes/blob/main/README.md#resources)
@jcanocan @nestoracunablanco Can you please confirm
```suggestion
* When xref:../../virt/monitoring/virt-exposing-downward-metrics.adoc#virt-configuring-downward-metrics_virt-exposing-downward-metrics[configuring a downward metrics device], the `spec.preference.name` value must be set to `rhel.9.s390x`.
```

Reply:
Looks like `rhel.9.s390x` is the correct preference name. 
I'm not an expert on x390x, and I don't have any s390x resource available to test. However, I don't see a reason for the downwardMetrics to just work on `RHEL 9`. This feature should work in any linux SO.

Reply:
According to the Readme and the [code](https://github.com/kubevirt/common-instancetypes/blob/main/preferences/rhel/9/s390x/kustomization.yaml#L14), the statement of @jcanocan and @SNiemann15 seems to be true. the downward metrics are currently supported only on RHEL 9 for the s390x architecture.

@jcanocan, in response to your remark: at this time, RHEL 9 is the only RHEL distribution enabled in common-instancetypes for s390x. I would suggest using *.s390x instead of rhel.9.s390x. One more question: does the downwardMetrics support regular expressions?

Reply:
To keep this 100% honest, there's no need either to use a preference, just a s390x linux SO.

> One more question: does the downwardMetrics support regular expressions?

Sorry, but I don't understand the question. Could you please elaborate? Here you can find the donwardMetrics documentation: https://kubevirt.io/user-guide/storage/disks_and_volumes/#downwardmetrics

Reply:
I have updated this to:
```
* When xref:../../virt/monitoring/virt-exposing-downward-metrics.adoc#virt-configuring-downward-metrics_virt-exposing-downward-metrics[configuring a downward metrics device], if you use a VM preference, the `spec.preference.name` value must be set to `rhel.9.s390x` or another available preference with the format `*.s390x`.
```

Please let me know if this is ok. Thanks!